backend/code_review_backend/issues/management/commands/cleanup_issues.py

```diff
@@ -29,24 +32,49 @@ def add_arguments(self, parser):
 
     def handle(self, *args, **options):
         clean_until = timezone.now() - timedelta(days=options["nb_days"])
+
+        # For issue and issuelink we can use _raw_delete since it's faster and doesn't
+        # give OOM since there are no CASCADE directives in the model since django
+        # doesn't need to query the model and compute the dependencies.
         rev_to_delete = Revision.objects.filter(
             base_repository__slug__in=["autoland", "mozilla-central"],
             head_repository__slug__in=["autoland", "mozilla-central"],
             created__lte=clean_until,
         )
+        count_rev = rev_to_delete.count()
 
-        count = rev_to_delete.count()
-        if not count:
+        if not count_rev:
             logger.info("Didn't find any old revision to delete.")
             return
 
-        # Delete revisions, with their related IssueLink and Diff
         logger.info(
-            f"Retrieved {count} old revisions from either autoland or mozilla-central to be deleted."
+            f"Retrieved {count_rev} old revisions from either autoland or mozilla-central to be deleted."
         )
-        _, stats = rev_to_delete.delete()
-        # Delete issues that are not linked to a revision anymore
-        _, issues_stats = Issue.objects.filter(revisions__isnull=True).delete()
-        stats.update(issues_stats)
+
+        stats = {}
+        for start in range(0, count_rev, DEL_CHUNK_SIZE):
+            # First fetch revisions IDs in a first DB request
+            chunk_rev_ids = rev_to_delete.order_by("id")[
+                start : start + UPDATE_CHUNK_SIZE
```

Comment:
```suggestion
                start : start + DEL_CHUNK_SIZE
```
We should keep a single variable here, otherwise it could cause some IDs to be skipped.